MdePkg/Test/Mock/Include/GoogleTest/Protocol/MockAcpiSystemDescriptionTable.h

```diff
@@ -0,0 +1,115 @@
+/** @file MockAcpiSystemDescriptionTable.h
+  This file declares a mock of ACPI system description tables protocol.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#ifndef MOCK_ACPI_SYSTEM_DESCRIPTION_TABLE_H
+#define MOCK_ACPI_SYSTEM_DESCRIPTION_TABLE_H

```

Comment:
For all header guards add the `_` postfix.

```suggestion
#ifndef MOCK_ACPI_SYSTEM_DESCRIPTION_TABLE_H_
#define MOCK_ACPI_SYSTEM_DESCRIPTION_TABLE_H_
```